x-pack/plugins/monitoring/server/lib/standalone_clusters/has_standalone_clusters.js

```diff
@@ -52,7 +52,7 @@ export async function hasStandaloneClusters(req, indexPatterns) {
   const { callWithRequest } = req.server.plugins.elasticsearch.getCluster('monitoring');
   const response = await callWithRequest(req, 'search', params);
   if (response && response.hits) {
-    return get(response, 'hits.total.value', 0) > 0;
+    return get(response, 'hits.total', 0) > 0 || get(response, 'hits.total.value', 0) > 0;
```

Comment:
Did this never work? The total hits should have always been at `hits.total`. If we want similar behavior, we should use `rest_total_hits_as_int`

Reply:
This is correct. The behavior never worked for 6.7 Kibana -> 6.7 ES

Reply:
Ok, then I still think this code should be changed because it still wouldn't work since it would always evaluate to false when running 7.0 due to `hits.total` being an object. We should add `rest_total_hits_as_int` to the request and only check for `hits.total`

Reply:
Or change this to a count request with `terminate_after: 1`

Reply:
@chrisdavies I don't think I have the ability to change the code - do you mind doing that? ^

Reply:
Will do.